content/docs/connectors/atlassian-crowd.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Authentication through Atlassian Crowd"
```

Comment:
```suggestion
title: "Authentication through Atlassian Crowd"
linkTitle: "Atlassian Cloud"
```

This should remove the "Auth through..." phrases on links (i.e., menu items.) it might require some modifications to the sidenav menu. feel free to ping me on slack as a follow-up.

Do for all pages in this section.

Reply:
👍